### PR TITLE
evalengine builtin func: CEIL

### PR DESCRIPTION
**Pull request by Jin-2019**
evalengine: Support all built-in MySQL functions CEIL

### CODE REVIEW

**Review by frouioui — CHANGES_REQUESTED** [review]
Hello @Jin-2019, thanks for your contribution! 🎉 

In order for the [DCO check](https://github.com/vitessio/vitess/pull/9795/checks?check_run_id=5353651422) to pass successfully, you will need to sign your commit by executing the following command within your forked repository:

```sh
git rebase HEAD~1 --signoff
git push --force-with-lease origin main
```

File: go/vt/vtgate/evalengine/integration/comparison_test.go [hunk 1]
```diff
@@ -572,3 +572,14 @@ func TestCharsetConversionOperators(t *testing.T) {
 		}
 	}
 }
+
+func TestCeil(t *testing.T) {
+	var conn = mysqlconn(t)
+	defer conn.Close()
+	var input = []string{
+		"0", "1", "1.123", "-1.23", "NULL",
```
Comment (vmg) [inline go/vt/vtgate/evalengine/integration/comparison_test.go:580]: You're not testing this with any floating point numbers. Could you please add some here? There are also other interesting test cases, such as strings, larger decimals, decimals closer to the `.5` half in rounding, larger negative decimals, numbers that would upcast their type when ceiled, etc. Please be imaginative and try to add as many test cases as possible.

**Review by vmg — CHANGES_REQUESTED** [review]
Thanks for your first contribution to Vitess! The behavior of the function you've implemented is not quite right. Please double check it and improve the integration tests. Cheers!

**Comment by systay** [comment]
Hi @Jin-2019!

Are you still interested in merging this PR? I'm going over old PRs checking what we can close.

**Pull request closed without merging**